docs/trace_service_features.md

```diff
@@ -2,34 +2,38 @@
 
 ## TOC
 
-- [Ad Service](#ad-service)
-- [Cart Service](#cart-service)
-- [Checkout Service](#checkout-service)
-- [Currency Service](#currency-service)
-- [Email Service](#email-service)
-- [Feature Flag Service](#feature-flag-service)
-- [Frontend Service](#frontend-service)
-- [Payment Service](#payment-service)
-- [Product Catalog Service](#product-catalog-service)
-- [Recommendation Service](#recommendation-service)
-- [Shipping Service](#shipping-service)
-
-## OpenTelemetry Feature Set by Services
+- [C++](#c)
+- [Erlang/Elxir](#erlangelixir)
+- [Go](#go)
+- [Java](#java)
+- [.NET](#net)
```

Comment:
Perhaps put this on top (`.` < `A`), or consider using `C#`

Reply:
If we use c# I think it’ll get confused with c++ since linking ignores special characters

Reply:
Maybe `#c-1`? (it's hard to maintain though)

Reply:
Could you commit an example of what you mean on the "." or point to an example

Reply:
Or we could use the trusty dotnet & cpp naming

Reply:
> Could you commit an example of what you mean on the "." or point to an example

I guess most developers would treat alphabetical ordering as "ordering based on the ASCII code or UNICODE ordinal".
So the order should be:
* .NET
* Algol
* Basic
* C++
* D
* Erlang
* Fortran
* Go

Reply:
In addition (I might be pedantic), I found it confusing when people mix up programming language and runtime. For example, I really think ECMAScript https://en.wikipedia.org/wiki/ECMAScript is the programming language (and the proverb is "JavaScript", which can be used to refer to ECMAScript or the very original language developed by Netscape, JScript developed by Microsoft or many other things).

I guess it's probably fine to use JavaScript as the name of the programming language, most developers would not pick whether it's ECMAScript or TypeScript.

I do think Node.js is NOT a programming language, it's a runtime.

Reply:
updated to dotnet and javascript